source/Calamari.Tests/Fixtures/Integration/Packages/NuGetPackageDownloaderFixture.cs

```diff
@@ -59,5 +63,76 @@ public int AttemptsTheRightNumberOfTimesOnError(int maxDownloadAttempts)
 
             return calledCount;
         }
+
+#if USE_NUGET_V2_LIBS
+        [Test]
```

Comment:
Consider adding a description to help people understand the background here
```suggestion
        [Test(Description="Fixes issue XXX")]
```